template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/s3/S3ConfigProvider.java

```diff
@@ -69,7 +71,8 @@ private void configureS3BucketLocationCoreSiteParameters(TemplatePreparationObje
                     .collect(Collectors.toSet());
             buckets.forEach(bucketName -> {
                 String s3BucketEndpointParam = String.format(S3_BUCKET_ENDPOINT_PARAM_TEMPLATE, bucketName);
```

Comment:
What about this bucker related endpoint param, it could remain the same endpoint?

Reply:
it should stay as is